src/status/CountryChart.js

```diff
@@ -44,6 +44,19 @@ const CountryChart = ({ t, peerLocations, className }) => {
     legend: {
       display: true,
       position: 'bottom'
+    },
+    tooltips: {
+      displayColors: false,
+      callbacks: {
+        title: (tooltipItem, data) => data.labels[tooltipItem[0].index],
+        label: (tooltipItem, data) => {
+          const percent = data['datasets'][0]['data'][tooltipItem['index']]
+          const count = Math.round((percent * totalCountries) / 100)
+          const peerStr = count === 1 ? t('peer') : t('peers')
```

Comment:
Does the translations module allow us to do plurals?

Reply:
Yes, it does. But as we already had those two keys I used those for convenience.

I change that though. What do you think?

Reply:
@fsdiogo better to use the i18next pluralisation magic as languages can have multiple plural forms depending on the number https://www.i18next.com/translation-function/plurals#languages-with-multiple-plurals

Reply:
Hmm, strangely I can't get that to work 🤔

```js
// public/locales/en/status.json

{
  ...
  "peersCount": "Peer",
  "peersCount_plural": "Peers",
  ...
}
```

```js
// src/status/CountryChart.js

const peerStr = t('peersCount', { count: count })
```

This should work, but it's always returning the singular form, even when the count is > 1.

Any ideas?

Reply:
We've configured i18next to use the ICU format for locales as it's a standard, rather than the default i18next format. See https://react.i18next.com/misc/using-with-icu-format